package.json

```diff
@@ -29,7 +29,8 @@
     "jstransform": "^11.0.0"
   },
   "devDependencies": {
-    "babel-core": "^5.2.7",
+    "babel": "^5.3.1",
+    "babel-core": "^5.3.1",
```

Comment:
Yea, I thought about it but couldn't find anything that actually said it was the same (without looking at src)